Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md

```diff
@@ -317,18 +317,23 @@ I know the ID of a customer that I want to delete.
       }
     }
 
-### Make RAW Request
+### Make Raw Request
+A simple action to allow integrators to assemble requests to be sent to the system.  The component should expose the parts that vary in a typical request.  The component should handle authentication and error reporting. It can optionally handle paging. In some cases, it may make sense to allow the option to make a series of sequential array requests.
 
-*This action has not been fully standardized.*
+##### Example Use Case
+I'm a technically advanced user who wants to interact with a system in a way not permissible by the existing component actions but would like some simplification relative to using the REST component.
 
-A simple action to allow integrators to assemble requests to be sent to the system.  The component should expose the parts that vary in a typical request.  The component should handle authentication and error reporting.
+##### Config Fields
+* Don't throw error on 404 responses (checkbox, optional): If checked, then the component should treat 404 response codes as success.
```

Comment:
Please don't limit it to 404.
How about a double array with a list of ranges. 
e.g. [[100, 199], [429, 429], [500, 599]]